src/asset-swapper/utils/market_operation_utils/index.ts

```diff
@@ -736,13 +771,19 @@ export class MarketOperationUtils {
                                   takerAddress: rfqt.takerAddress,
                                   takerToken,
                                   txOrigin: rfqt.txOrigin,
-                              })
-                          ).quotes.map((quote) => {
-                              DEFAULT_INFO_LOGGER({ ...quote, txOrigin: rfqt.txOrigin }, 'results from RFQ Client');
-                              // HACK: set the signature on quoteRequestor for future lookup (i.e. in Quote Report)
-                              rfqt.quoteRequestor?.setMakerUriForSignature(quote.signature, quote.makerUri);
-                              return toSignedNativeOrder(quote);
-                          });
+                              }),
+                          ]);
+
+                if (v2Quotes.length > 0) {
```

Comment:
Shall we log the queues even if there are non of them, for completeness and clearness?